squidb-processor/src/com/yahoo/squidb/processor/plugins/defaults/properties/generators/BasicPropertyGenerator.java

```diff
@@ -67,8 +67,6 @@ public BasicPropertyGenerator(ModelSpec<?> modelSpec, VariableElement field, Apt
     private void validateColumnName() {
         if (columnName.indexOf('$') >= 0) {
```

Comment:
I wonder if we should move this to `checkIdentifier()` so it's enforced on tables and views, too.